src/style.css

```diff
@@ -1,7 +1,70 @@
+*,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
 body {
-  background-color: bisque;
+  background-color: white;
+}
+
+h1 {
+  padding: 10px;
+  text-align: center;
```

Comment:
- In the original wireframe design, the title is aligned left. Kindly stick with the original design to have the minimum styling. 
![image](https://user-images.githubusercontent.com/64894927/131400921-faa6d378-9e42-4fe3-b870-ce40bf885718.png)
![image](https://user-images.githubusercontent.com/64894927/131400978-15202d81-29e4-4fae-b716-edbf95454686.png)

Reply:
This issue is well resolved! ✔️